README.org

```diff
@@ -111,8 +111,8 @@ By default =emacs-plus@29= uses the following features.
 
 *** Options
 
-| Option                    | Description                                                                  |
-|---------------------------+------------------------------------------------------------------------------|
+| Option                  | Description                                                                  |
+|-------------------------+------------------------------------------------------------------------------|
```

Comment:
Your alignment ignores `=` symbols :) 

![image](https://user-images.githubusercontent.com/6507913/186158104-2bbf0a0c-11f0-4b2a-9e77-eaa331982724.png)

Reply:
Removed the alignment, sorry 🤦‍♂️